src/assets/scripts/client/commands/parsers/CommandParser.js

```diff
@@ -128,7 +128,7 @@ export default class CommandParser {
      */
     _extractCommandsAndArgs(rawCommandWithArgs) {
         const commandOrCallsignIndex = 0;
-        const commandArgSegmentsWithCallsign = rawCommandWithArgs.split(COMMAND_ARGS_SEPARATOR);
+        const commandArgSegmentsWithCallsign = rawCommandWithArgs.split(COMMAND_ARGS_SEPARATOR).filter((t) => t);
```

Comment:
Is this coercing an empty string (space) to falsey and filtering those out? I've never seen this before. If that's it, this is clever.